search.js

```diff
@@ -3,7 +3,7 @@ const books = require('google-books-search');
 const Promise = require('bluebird')
 const _ = require('lodash')
 
-const SEARCH_LIMIT = 9
+const SEARCH_LIMIT = 36
```

Comment:
you could make 4 calls (4*36 = 144 books) and return the entire resultset to the client and let it paginate

Reply:
Not going to make changes here yet as I'm not sure how to do either. I spent some time researching and wasn't able to find enough information to assist. If you have time, can we look at it together?

Reply:
Definitely